src/main/java/org/launchcode/mentalhealthcareaccess/models/data/dto/RegisterFormDTO.java

```diff
@@ -0,0 +1,35 @@
+package org.launchcode.mentalhealthcareaccess.models.data.dto;
+
+import javax.validation.constraints.NotBlank;
+
+public class RegisterFormDTO extends LoginFormDTO {
+    @NotBlank
+    private String verifyPassword;
+    @NotBlank
+    private String name;
```

Comment:
A better name for this field/prop is probably something like `firstName`. Otherwise, it may raise the question of how `name` and `lastName` are related.

Reply:
Actually, I see now that `name` may be doing double duty as a company name OR an individual's first name. Do I have that right? If so, it may make more sense to have separate fields with some more complex validation logic. Then you could use a "derived property" (i.e. a field that doesn't have an associate property but includes logic to generate the value). 

For instance, I'm thinking of the case of displaying providers in alphabetical order. For a company, you'd want to use the company name, but for an individual you'd want to use the last name. Currently, these are in separate fields.

In any case, we can chat about this during the project check-in.